cmd/main.go

```diff
@@ -293,6 +294,7 @@ func main() { // nolint:gocyclo
 		Client:         mgr.GetClient(),
 		GetMembersFunc: commoncluster.GetMemberClusters,
 		Scheme:         mgr.GetScheme(),
+		FieldManager:   hostOperatorFieldManager,
```

Comment:
do we need to store it as a field of the Reconciler type? Couldn't we move it as an exported (global) constant and use it directly when creating the SSA client? The field is not set in the unit tests at all, so using the constant should work fine.